.changeset/stupid-rockets-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/reactive-element': patch
+---
+
+Fix `@queryAssignedElements` decorator so internal implementation is compatible with Lit's browser support matrix.
```

Comment:
Could rephrase this to be a little more useful to users -- specifically it fixes the decorator for a certain range of Firefox versions, right?

Reply:
Sorry the Firefox version stuff was a complexity when deciding where to put the `assignedElements` "polyfill". Either in Lit directly, or in ShadyDOM.

The Firefox version in the support matrix means for full legacy browser & modern browser support, we should put the polyfill in the decorator.

I'll update the changelog to be more straightforward. LMK if it's better. Thank you!!!